packages/scheduler/lib/models/tasks.integration.test.ts

```diff
@@ -113,20 +110,23 @@ describe('Task', () => {
         await new Promise((resolve) => void setTimeout(resolve, timeout * 1100));
         const expired = (await tasks.expiresIfTimeout()).unwrap();
         expect(expired).toHaveLength(1);
+        expect(expired[0]?.output).toMatchObject({ reason: `Timeout 'createdToStartedTimeoutSecs' exceeded` });
```

Comment:
just a comment not a suggestion, usually I prefer storing error code (along with  details if necessary) instead of full sentence it's more portable and easy to filter